ganttproject/src/net/sourceforge/ganttproject/GanttProject.java

```diff
@@ -665,11 +663,7 @@ void doShow() {
       Class.forName("java.awt.desktop.AboutHandler");
```

Comment:
Since we're targeting at Java 11, I suggest removing this line and the surrounding try/catch block too. It was needed to be able to run under both Java 8 and Java 11 (there were no AboutHandler in Java 8), but now it is obsolete.

Reply:
Yes, I tested GP in macOS Catalina after removing it and it works as expected. It's great to help cleaning unused/unneeded lines of code :)

![image](https://user-images.githubusercontent.com/1078305/75233749-a5c5ee80-57b9-11ea-85af-e9eb74e6d166.png)